src/Database/LSMTree/Normal.hs

```diff
@@ -32,13 +32,13 @@ module Database.LSMTree.Normal (
   , Common.MergeTrace (..)
 
     -- * Table sessions
-  , Session
+  , Session (..)
   , withSession
   , openSession
   , closeSession
 
     -- * Table handles
-  , TableHandle
+  , TableHandle (..)
```

Comment:
I'm not sure why my comment got attached to the removal rather than the change.

To be clear, it's the exporting of Session and TableHandle (..) representations that I'm grumbling about.
```suggestion
  , TableHandle
```